usefull_app/common/tests.py

```diff
@@ -63,3 +65,49 @@ def add_passenger_and_permission_token_to_db() -> tuple[Passenger, Token]:
         user, token = TestMixin.add_user_and_permission_token_to_db()
         passenger = Passenger.objects.create(user=user)
         return passenger, token
+
+    @staticmethod
+    def add_rated_passenger_to_db() -> tuple[Passenger, PassengerRating]:
+        """
+        Create and return Passenger, with PassengerRating entries added to return entry.
+        """
+        passenger = TestMixin.add_passenger_to_db()
+        rated_by_users = [TestMixin.add_random_users_to_db() for i in range(2)]
+        for number, user in enumerate(rated_by_users):
+            if number % 2 == 0:
+                rate = 5
+            else:
+                rate = 1
+            rate_data = {
+                'user': user,
+                'passenger': passenger,
+                'rate': rate,
+            }
+            rate = PassengerRating.objects.create(**rate_data)
```

Comment:
There's no need to create a dict and then unpack it.
```suggestion
            rate = PassengerRating.objects.create(
            user=user,
            passenger=passenger,
            rate=5 if number % 2 == 0 else 1,
            )
```

Reply:
Changed in new commit.